src/Styleguide/Components/__stories__/SelectedExhibitions.story.tsx

```diff
@@ -11,28 +11,33 @@ storiesOf("Styleguide/Components", module).add("SelectedExhibitions", () => {
   return (
     <React.Fragment>
       <Section title="3 exhibitions responsive">
-        <SelectedExhibitions exhibitions={exhibitions} />
+        <SelectedExhibitions exhibitions={exhibitions as any} />
```

Comment:
Why are the `as any` statements needed?

Reply:
Still need to go over and type everything in a later sweep. Didn’t want to lock ourselves into a structure that might change as patterns settle.